### ISSUE TITLE
is resnet_v2_101/logits the same as fc1000?

### ISSUE THREAD

**Issue by ghost** [tomrunia/TF_FeatureExtraction#14]
Hi, 
I am wondering is resnet_v2_101/logits the same as fc1000 in the below architecture
http://ethereon.github.io/netscope/#/gist/b21e2aae116dc1ac7b50
